sbin/celeborn-daemon.sh

```diff
@@ -74,36 +74,34 @@ celeborn_rotate_log ()
     log=$1;
     num=5;
     if [ -n "$2" ]; then
-	num=$2
+	    num=$2
     fi
     if [ -f "$log" ]; then # rotate logs
-	while [ $num -gt 1 ]; do
-	    prev=`expr $num - 1`
-	    [ -f "$log.$prev" ] && mv "$log.$prev" "$log.$num"
-	    num=$prev
-	done
-	mv "$log" "$log.$num";
+	    while [ $num -gt 1 ]; do
+	      prev=`expr $num - 1`
+	      [ -f "$log.$prev" ] && mv "$log.$prev" "$log.$num"
+	      num=$prev
```

Comment:
Code behave differently in local and Github.

![WX20221129-144133](https://user-images.githubusercontent.com/52876270/204457488-75b5fa8b-e435-42ce-a505-4d4d2f03591b.png)

Reply:
Seems there are tabs in the beginning of the line, we can replace with whitespace

Reply:
![image](https://user-images.githubusercontent.com/948245/204470774-b117a23b-0aaf-4b4c-8f5f-b5c0123d3b0d.png)

Reply:
done